hoot-test/src/main/cpp/hoot/test/main.cpp

```diff
@@ -419,7 +419,7 @@ int main(int argc, char *argv[])
       while (testName != HOOT_TEST_FINISHED)
       {
         rootSuite = new CppUnit::TestSuite( "All tests" );
-        populateTests(ALL, rootSuite, printDiff);
+        populateTests(ALL, rootSuite, printDiff, true);
```

Comment:
Hide the "Disable" warnings on the slave processes because they are already printed out on the master process.